try_bs.py

```diff
@@ -0,0 +1,22 @@
+import requests
```

Comment:
please remove files not related to labs implementation

Reply:
fixed